docs/developer-guide/rules.md

```diff
@@ -28,9 +28,19 @@ We take care to ensure that all the rules are named accurately and consistently.
 
 ### Determine options
 
-Every rule must have a **primary option**. In `"color-hex-case": "upper"`, the primary option is `"upper"`; in `"indentation": [2, { "except": ["block"] }]`, the primary option is `2`.
+#### Primary
 
-Some rules require extra flexibility to address a variety of use-cases. These can use a **secondary options object**. In `"color-hex-case": "upper"`, there is no secondary options object; in `"indentation": [2, { "except": ["block"] }]`, the secondary options object is `{ "except": ["block"] }`.
+Every rule *must have* a **primary option**.
+
+- In `"color-hex-case": "upper"`, the primary option is `"upper"`
+- In `"indentation": [2, { "except": ["block"] }]`, the primary option is `2`.
+
+#### Secondary
+
+Some rules require extra flexibility to address a variety of use-cases. These can use a **secondary options object**.
+
+- In `"color-hex-case": "upper"`, there is no secondary options object
```

Comment:
Added full stops in https://github.com/stylelint/stylelint/commit/81cfb916c6ba50d487dd6108d75c6565a030a9a7